bindings/py/src/test/bindings/py/tests/algorithms/cells4_test.py

```diff
@@ -1,290 +0,0 @@
-          print "Num segments different in cell:", c, i,
-          print "numbers = ", cell1.nSegmentsOnCell(c, i), \
-              cell2.nSegmentsOnCell(c, i)
-          result = False
-          break
-        else:
-          c1 = cell1.getCell(c, i)
-          c2 = cell2.getCell(c, i)
-          for j in xrange(cell1.nSegmentsOnCell(c, i)):
-            seg1 = c1.getSegment(j)
-            seg2 = c2.getSegment(j)
-            if seg1.size() != seg2.size():
-              result = False
-              break
-            for k in xrange(seg1.size()):
-              sourceCellIdx1 = seg1.getSrcCellIdx(k)
-              sourceCellIdx2 = seg1.getSrcCellIdx(k)
-              if sourceCellIdx1 != sourceCellIdx2:
-                result = False
-                break
-              perm1 = seg1.getPermanence(k)
-              perm2 = seg2.getPermanence(k)
-              if perm1 != perm2:
-                result = False
-                break
-
-    if result == True:
-      print "TP's match"
-
-    return result
-
-
-  def _testPersistence(self, cells):
-    """This will pickle the cells instance, unpickle it, and test to ensure
-    the unpickled instance is identical to the pre-pickled version.
-    """
-    file1 = "test.pkl"
-    file2 = "test2.bin"
-    pickle.dump(cells, open(file1, "wb"))
-    cells2 = pickle.load(open(file1))
-
-    # Test all public attributes of Cells4 that should get pickled
-    for f1, f2 in zip(dir(cells), dir(cells2)):
-      if f1[0] != "_" and f1 not in ["initialize", "setStatePointers",
-                                     "getStates", "rebuildOutSynapses"]:
-        ff1, ff2 = getattr(cells, f1), getattr(cells, f2)
-        try:
-          r1, r2 = ff1(), ff2()
-          resultsEqual = (r1 == r2)
-        except (NotImplementedError, RuntimeError, TypeError, ValueError):
-          continue
-        self.assertTrue(resultsEqual, "Cells do not match.")
-
-    # Ensure that the cells are identical
-    self.assertTrue(self._cellsDiff(cells, cells2))
-
-    os.unlink(file1)
-
-    # Now try the Cells4.saveToFile method.
-    cells.saveToFile(file2)
-    cells2 = Cells4()
-    cells2.loadFromFile(file2)
-
-    self.assertTrue(self._cellsDiff(cells, cells2))
-
-    # Test all public attributes of Cells4 that should get pickled
-    for f1, f2 in zip(dir(cells), dir(cells2)):
-      if f1[0] != "_" and f1 not in ["initialize", "setStatePointers",
-                                     "getStates", "rebuildOutSynapses"]:
-        ff1, ff2 = getattr(cells, f1), getattr(cells, f2)
-        try:
-          r1, r2 = ff1(), ff2()
-          resultsEqual = (r1 == r2)
-        except (NotImplementedError, RuntimeError, TypeError, ValueError):
-          continue
-        self.assertTrue(resultsEqual, "Cells do not match.")
-
-    # Ensure that the cells are identical
-    self.assertTrue(self._cellsDiff(cells, cells2))
-
-    os.unlink(file2)
-
-
-  def testLearn(self):
```

Comment:
why can this file for Cells4 be deleted?

Reply:
> why can this file for Cells4 be deleted?

It was not deleted, The entire tree at `bindings/py/src/test/bindings/py/tests` was moved to `bindings/py/src/tests`